lib/discordrb/bot.rb

```diff
@@ -339,7 +339,7 @@ def voice_connect(chan, encrypted = true)
 
     # Disconnects the client from a specific voice connection given the server ID. Usually it's more convenient to use
     # {Discordrb::Voice::VoiceBot#destroy} rather than this.
-    # @param server [Server, Integer, #resolve_id] The server the voice connection is on.
+    # @param server [Server, String, Integer] The server, or server id, the voice connection is on.
```

Comment:
Capitalize it consistently reeeee
```suggestion
    # @param server [Server, String, Integer] The server, or server ID, the voice connection is on.
```